scipy/interpolate/__init__.py

```diff
@@ -5,87 +5,117 @@
 
 .. currentmodule:: scipy.interpolate
 
-Sub-package for objects used in interpolation.
+Sub-package for functions and objects used in interpolation.
+
+See the `user guide <tutorial-interpolate>` for recommendations on choosing a routine,
+and other usage details.
 
-As listed below, this sub-package contains spline functions and classes,
-1-D and multidimensional (univariate and multivariate)
-interpolation classes, Lagrange and Taylor polynomial interpolators, and
-wrappers for `FITPACK <http://www.netlib.org/dierckx/>`__
-and DFITPACK functions.
 
 Univariate interpolation
 ========================
 
 .. autosummary::
    :toctree: generated/
 
-   interp1d
-   BarycentricInterpolator
-   KroghInterpolator
-   barycentric_interpolate
-   krogh_interpolate
-   pchip_interpolate
-   CubicHermiteSpline
+   make_interp_spline
+   CubicSpline
    PchipInterpolator
    Akima1DInterpolator
-   CubicSpline
+   FloaterHormannInterpolator
+   BarycentricInterpolator
+   KroghInterpolator
+
+Low-level data structures for univariate interpolation
+------------------------------------------------------
+
+.. autosummary::
+   :toctree: generated/
+
    PPoly
    BPoly
-   FloaterHormannInterpolator
+   BSpline
+   CubicHermiteSpline
```

Comment:
Can we put this at the top? Although this happens to be the base class of some of the others, it's pretty thin on top of `PPoly` and quite usable on its own. I'd say the criterion for "low-level" is that these others don't accept `x`, `y`.

Reply:
Yup, this can go either way. Done and force-pushed (+a couple of fix-ups for the layout and references).